onnxruntime/test/framework/execution_frame_test.cc

```diff
@@ -65,7 +65,7 @@ TEST_F(ExecutionFrameTest, TensorAllocationTest) {
 
   std::unique_ptr<SequentialExecutionPlan> p_seq_exec_plan;
   // TODO below line is for testing only. In production use SequentialPlanner::CreatePlan()
-  SequentialPlannerContext context(ExecutionMode::ORT_SEQUENTIAL);
+  SequentialPlannerContext context(ExecutionMode::ORT_SEQUENTIAL, ExecutionOrder::TOPOLOGICAL);
```

Comment:
>ExecutionOrder::TOPOLOGICAL [](start = 66, length = 27)

Should we add some tests for priority based execution order?

Reply:
done.